From db6157bd3d21b1e807fe918e5a352b216660d548 Mon Sep 17 00:00:00 2001
From: Nirbheek Chauhan <nirbheek@centricular.com>
Date: Wed, 17 Apr 2019 18:54:26 +0530
Subject: [PATCH] meson: Add a quick-hack to support VS2019

Proper support is already upstream in 0.50.1, but it's too close to
the 1.16 release to bump to that.
---
 mesonbuild/compilers/c.py | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/mesonbuild/compilers/c.py b/mesonbuild/compilers/c.py
index 655dfb3..d47694e 100644
--- a/mesonbuild/compilers/c.py
+++ b/mesonbuild/compilers/c.py
@@ -1563,6 +1563,8 @@ class VisualStudioCCompiler(CCompiler):
             return '14.0' # (Visual Studio 2015)
         elif version < 1920:
             return '14.1' # (Visual Studio 2017)
+        elif version < 1930:
+            return '14.2' # (Visual Studio 2017)
         return None
 
     def get_default_include_dirs(self):
-- 
2.20.1.windows.1

